src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -220,6 +232,15 @@ private IList<ManifestItem> FindTocInManifest(Manifest manifest)
             return manifest.Files.Where(f => IsType(f, ManifestItemType.Toc)).ToList();
         }
 
+        private ManifestItem FindSiblingCoverPageInManifest(Manifest manifest, ManifestItem tocFile)
+        {
+            return manifest.Files.SingleOrDefault(f =>
+            {
+                return Path.GetDirectoryName(f.SourceRelativePath).StartsWith(Path.GetDirectoryName(tocFile.SourceRelativePath))
```

Comment:
Consider 2 folder names like `pdf` and `pdf1`, `StartsWith` is not safe. This line should be more strict like `Path.GetDirectoryName(f.SourceRelativePath) == Path.GetDirectoryName(tocFile.SourceRelativePath)`